src/components/SizeTile.svelte

```diff
@@ -19,21 +19,6 @@ $: alt = name ? `picture of something ${name}` : ''
 </script>
 
 <style>
-/* Technique based on https://stackoverflow.com/a/17541916 */
-[type=radio]:focus + .btn-outline-dark {
-  border-color: var(--blue);
-  color: var(--blue);
-}
-
-[type=radio]:focus:hover + .btn {
-  border-color: var(--dark);
-  color: var(--light);
-}
-
-[type=radio]:hover + .btn {
-  background-color: var(--blue);
-}
-
```

Comment:
These are to keep both mouse interactions and keyboard interactions intuitive, so if you do decide to remove them, make sure navigating among the SizeTiles using a keyboard is still clear (selected vs. currently-has-focus).

Reply:
It's not totally clear (no size is selected when tabbed into the component). As it tends to go, for me at least, css things like this can chew up a lot of time. I'll make a TODO note and move on for now. Chances are good this will be redesigned anyway.